pom.xml

```diff
@@ -87,6 +88,24 @@
 		</dependency>
 
 
+		<!---->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-security</artifactId>
+		</dependency>
+
+		<dependency>
```

Comment:
Acredito que é possível configurar o protocolo OAuth neste projeto sem utilizar esta lib.

https://docs.spring.io/spring-security-oauth2-boot/docs/2.0.x/reference/htmlsingle/

`This project is a port of the Spring Security OAuth support that came with Spring Boot 1.x. Support was removed in favor of Spring Security 5’s first class OAuth support. To ease migration, this project exists as a bridge between the old Spring Security OAuth support and Spring Boot 2.x.`
--

Reply:
Verdade Henrique, obrigado pelo feedback. Já ajustei.